plugins/image-source/data/locale/en-US.ini

```diff
@@ -29,6 +29,7 @@ SlideShow.Stop="Stop"
 SlideShow.NextSlide="Next Slide"
 SlideShow.PreviousSlide="Previous Slide"
 SlideShow.HideWhenDone="Hide when slideshow is done"
+SlideShow.MaxMemoryUsage="Maximum Memory Usage (megabytes)"
```

Comment:
```suggestion
SlideShow.MaxMemoryUsage="Maximum VRAM Usage"
```

Reply:
Maybe even better: "Maximum Graphics Memory Usage"?

I think "VRAM" is an abbreviation that might already be confusing to non-programmers/non-gamers.

Reply:
gotcha, modified.

Reply:
"Maximum GPU Memory Usage" sounds clearer to me. "Graphics" is pretty abstract, while this directly refers to the hardware in question, just like you would say "CPU Usage" and not "Compute usage".

Reply:
👍 updated to "Maximum GPU Memory Usage"